ibis/backends/dask/tests/conftest.py

```diff
@@ -54,15 +54,30 @@ def assert_series_equal(
         kwargs.setdefault('check_dtype', cls.check_dtype)
         kwargs.setdefault('check_names', cls.check_names)
         # we sometimes use pandas to build the "expected" case in tests
-        right = right.compute() if isinstance(right, dd.Series) else right
-        tm.assert_series_equal(left.compute(), right, *args, **kwargs)
+        incoming_types = tuple(map(type, (left, right)))
+        if incoming_types == (dd.Series, pd.Series):
```

Comment:
Because I only want to do a `reset_index` behavior if the left one was created in dask and has more than one partition, while the right one was created in pandas. In all other circumstances we can just `compute()` any dask output.